pkg/plugins/manager/loader/initializer/initializer.go

```diff
@@ -51,6 +51,9 @@ func (i *Initializer) envVars(plugin *plugins.Plugin) []string {
 	hostEnv := []string{
 		fmt.Sprintf("GF_VERSION=%s", i.cfg.BuildVersion),
 	}
+	if plugin.Info.Version != "" {
+		hostEnv = append(hostEnv, fmt.Sprintf("GF_PLUGIN_VERSION=%s", plugin.Info.Version))
```

Comment:
Not sure about this one. Personally, I would prefer that the backend plugin reads plugin.json at startup to get the plugin's version (and same for the plugin id, to simplify the call to `datasource.Manage` for example), but that's out of scope for this PR I guess